entrypoint_scripts/os/linux.sh

```diff
@@ -320,6 +329,14 @@ function install_linux() {
         echo "  Installing database on CentOS"
 		echo "  TBD: DB install for CentOS"
 		;;
+	    "Linux Mint")
```

Comment:
As Ubuntu and Linux Mint are handled the same way here, this structure creates code duplication, making it harder for maintenance.
```suggestion
	    "Ubuntu" | "Linux Mint")
```